infrastructure/oss/issue.go

```diff
@@ -45,8 +46,23 @@ func toIssue(
 	ep error_reporting.ErrorReporter,
 ) snyk.Issue {
 	// this needs to be first so that the lesson from Snyk Learn is added
-	codeActions := issue.AddCodeActions(learnService, ep)
-
+	codeActions := issue.AddCodeActions(learnService, ep, affectedFilePath, issueRange)
+
+	var codelensCommands []snyk.CommandData
+	for _, codeAction := range codeActions {
+		fmt.Println(codeAction.Title)
+		if strings.Contains(codeAction.Title, "Upgrade to") {
+			codelensCommands = append(codelensCommands, snyk.CommandData{
+				Title:     "⚡ Fix this issue: " + codeAction.Title,
+				CommandId: snyk.CodeFixCommand,
```

Comment:
Interesting solution to use the Autofix command. Wouldn't have thought of it, but it's great!